packages/app/src/Main.tsx

```diff
@@ -1,14 +1,13 @@
-import { noop } from 'lodash'
 import { View } from 'native-base'
 import { useEffect } from 'react'
 
 import { FFMapView } from './components/FFMapView'
 import { UIOverlay } from './components/UIOverlay'
-import { client } from './tracking'
+import { track } from './tracking'
 
 export const Main = () => {
     useEffect(() => {
-        client.hit().catch(noop)
+        track({ name: 'app-opened' })
```

Comment:
I think instead of sending this as an event you can send it as a `/hit` request and attach some info about the device that way it is closer to the real sdk and you will have a better dashboard

Reply:
https://docs.pirsch.io/api-sdks/api-guide

Reply:
good point